skimage/io/tests/conftest.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+import numpy as np
+import imageio
+
+
+@pytest.fixture(scope="session")
+def random_gif_path(tmpdir_factory):
+    """Create "random.gif" once per session and return its path."""
+    rng = np.random.default_rng(42)
+    img = rng.integers(0, 255, (24, 25, 14, 3), dtype=np.uint8)
+    tmp_path = str(tmpdir_factory.mktemp("session-data").join("random.gif"))
```

Comment:
In this case I can't use `tmp_file` because it's function scoped while I created this fixture with a session scope and this is how [pytest's docs suggest to do this](https://docs.pytest.org/en/7.3.x/how-to/tmp_path.html#the-tmp-path-factory-fixture). It's till good that you commented though, because I'm using the old `tmpdir_factory` here but should probably use `tmp_path_factory.mktemp` instead, which returns a `Path` object.